src/components/Table/Table.module.css

```diff
@@ -0,0 +1,12 @@
+.table {
+  border-collapse: collapse;
+  font-size: 9px;
+  text-align: center;
+  width: 100%;
+  table-layout: fixed;
+}
+.table td,
```

Comment:
🤔 I'm usually cautious with element selectors ( I know it's in a css module ).

maybe create very small components like

```tsx
const TableRow: React.FC<{children: ReactNode, className?: string}> = (props) => (
    <tr {...props} className={classNames(styles.tr, props.className)} />
)
```